docs/using-the-compiler.rst

```diff
@@ -29,6 +29,8 @@ files reside, so things like ``import "/etc/passwd";`` only work if you add ``=/
 
 If there are multiple matches due to remappings, the one with the longest common prefix is selected.
 
+Important to note is that paths of included sources files and paths defined by remappings are allowed to include, but everything else is rejected. Additional paths can be added via the ``--allow-paths /sample/path,/another/sample/path`` switch.
```

Comment:
`paths of source files specified on the commandline and paths defined by remappings...`

It is probably better to put that in line 16.